app/helpers/index.ts

```diff
@@ -36,4 +38,46 @@ export const duration = (seconds: number, t: (str: string) => string) => {
   return `${sec}${t('labels.sec')}`;
 };
 
+export const timeSince = (timeStamp: number, t: TFunction) => {
+  const secPerMinute = 60;
+  const secPerHour = secPerMinute * 60;
+  const secPerDay = secPerHour * 24;
+  const secPerMonth = secPerDay * 30;
+  const secPerYear = secPerDay * 365;
+  const now = new Date();
+  const seconds = Math.floor(now.getTime() / 1000 - timeStamp);
```

Comment:
Можно не создавать инстанс
```suggestion
  const seconds = Math.floor(Date.now() / 1000 - timeStamp);
```